edg/parts/Tle5012b.py

```diff
@@ -0,0 +1,77 @@
+from edg.abstract_parts import *
+from edg.parts.JlcPart import JlcPart
+
+
+class Tle5012b_Device(InternalSubcircuit, FootprintBlock, JlcPart):
+    def __init__(self) -> None:
+        super().__init__()
+        self.vdd = self.Port(VoltageSink(
+            voltage_limits=(3.0, 5.5) * Volt,
+            current_draw=(5 * uAmp, 16 * mAmp)
+        ))
+        self.gnd = self.Port(Ground())
+
+        dio_model = DigitalBidir.from_supply(
+            self.gnd, self.vdd,
+            voltage_limit_tolerance=(-0.3, 0.3) * Volt,
+            input_threshold_factor=(0.3, 0.7),
+            current_limits=(5 * uAmp, 25 * mAmp)
```

Comment:
Where'd this come from?
It seems that it can sink up to -25mA and has a pull-up rating of 3uA? I don't see a source current limit even though there's a section on the SSC interface in push-pull configuration.